.gitmodules

```diff
@@ -16,6 +16,6 @@
 [submodule "tools/zachjs-sv2v"]
 	path = tools/zachjs-sv2v
 	url = https://github.com/zachjs/sv2v
-[submodule "external/swerv"]
```

Comment:
With the above comment about `third_party/tools/zachjs-sv2v` this should probably end up as something like `third_party/cores/swerv` or `third_party/tests/swerv` or similar?